File: BJ8958/main.c
//
//  OX퀴즈
//  BJ8958
//
//  Created by 홍준우 on 6/29/24.
//

#include <stdio.h>

int main(void) {
    int n, tmp, score;
    char ch[81];
    scanf("%d", &n);
    for (int i = 0; i < n; i++) {
        score = tmp = 0;
        scanf("%s", ch);
        for (int i = 0; ch[i] != '\0'; i++) {
            if (ch[i] == 'O') score += ++tmp;
            else tmp = 0;
        }
        printf("%d\n", score);
    }
    return 0;
}

File: BJ8958/main.py
#
#  OX퀴즈
#  BJ8958
#
#  Created by 홍준우 on 6/30/24.
#

for _ in range(int(input())):
    score = tmp = 0
    for ch in input():
        if ch == 'O':
            tmp += 1
            score += tmp
        else: tmp = 0
    print(score)
